atlp-rwanda/winners-c8-bn-be
# Fix hound CI,  eslint and formatting

#### What does this PR do?
- make sure that Hound CI does not throw error on PR
- add the pull_request_template with some notes on the bottom
- add the script to fix fixable eslint errors `npm run fix`

#### Description of Task to be completed?
- make sure that the code can properly be reviewed
#### How should this be manually tested?
-  when you run `npm run lint` should shows errors when eslint rules are not followedd
#### Any background context you want to provide?
- the hound ci not properly reviewing using eslint so I have integrated the github action to check  the eslint errors
#### What are the relevant pivotal tracker stories?
#182209988
#182209987
#### Screenshots (if appropriate)
#### Questions:
